### PR TITLE
(maint) Add require for Open3 module

### PR DESCRIPTION
**Pull request by Magisus**
This require appears to be necessary for running in CI, though it wasn't
needed when testing the rake task locally.

### CODE REVIEW

**Comment by smcclellan** [comment]
Why are we removing Gemfile.lock?

**Comment by Magisus** [comment]
It's ignored in our other Ruby stuff, so I thought that it was supposed to be, but no one had made a gitignore for this gem. Did you have a reason for keeping it checked into Git?

**Comment by Magisus** [comment]
Which is interesting, because reading more about it, it looks like it's recommended to check it in. Wonder why we don't in general.

**Comment by smcclellan** [comment]
It's a way to specify the exact version of a gem that is tested and being used. In many projects we have open-ended dependencies, and this is a way to require the user to run `bundle update` before checking a later dependency which might break existing functionality.

**Comment by Magisus** [comment]
Okay well I can remove it from the gitignore :)

**Comment by smcclellan** [comment]
Well... I guess it's different when you're working on a gem: http://stackoverflow.com/a/4151540/7513050

**Comment by smcclellan** [comment]
*shrug* I guess we can remove it.

File: lib/tasks/gettext.rake [hunk 1]
```diff
@@ -62,6 +62,7 @@ namespace :gettext do
       old_pot = pot_file_path + ".old"
       File.rename(pot_file_path, old_pot)
       generate_new_pot
+      require 'open3'
```
Comment (smcclellan) [inline lib/tasks/gettext.rake]: I'd require this at the top of the `else` block, before the renaming/copying happens, in case the require fails.

Comment (Magisus) [inline lib/tasks/gettext.rake]: That makes sense. Updated.

**Comment by Magisus** [comment]
Do you have any idea why this was working locally without the require, but not on Jenkins?

**Comment by smcclellan** [comment]
Is it being run through bundler, locally and on Jenkins?

**Comment by Magisus** [comment]
Yes. @branan theorized different Ruby versions might pre-load different things.

**Comment by smcclellan** [comment]
Could be. I'm curious which versions include/don't include it.

**Comment by ScottGarman** [comment]
:+1:

**Comment by branan** [comment]
I can't merge this with amerge commit (this repo is set to only allow rebase and merge?) I'm looking in to that now, as that's different from our normal merge-commit-based workflow

File: lib/tasks/gettext.rake [hunk 2]
```diff
@@ -59,6 +59,7 @@ namespace :gettext do
       puts "No existing POT file, generating new"
       generate_new_pot
     else
+      require 'open3'
```
Comment (MosesMendoza) [inline lib/tasks/gettext.rake]: I think its customary for all require statements to be at the top of the file

Comment (Magisus) [inline lib/tasks/gettext.rake]: Alright 👍  I wasn't sure what the policy was for rake tasks.

**Comment by smcclellan** [comment]
@branan Rebase and merge is by design for this repository, as it's a mergeless repo (much cleaner git log, easier for rebasing, etc.). Big green button is the workflow. 👍

**Pull request merged**